server/php-master.cpp

```diff
@@ -1536,6 +1536,8 @@ STATS_PROVIDER_TAGGED(kphp_stats, 100, STATS_TAG_KPHP_SERVER) {
   add_histogram_stat_double(stats, "requests.incoming_queries_per_second", qps_calculator.get_incoming_qps());
   add_histogram_stat_double(stats, "requests.outgoing_queries_per_second", qps_calculator.get_outgoing_qps());
 
+  add_histogram_stat_double(stats, "graceful_restart.warmup.final_instance_cache_sizes_ratio", WarmUpContext::get().calc_final_instance_cache_sizes_ratio());
```

Comment:
Let's write both values instead of ratio. so we can catch the case if `final_old_instance_cache_size`  is small.

Reply:
done